docs/docs/aws.md

```diff
@@ -428,7 +428,7 @@ For example, to add S3 delete tags with Spark 3.3, you can start the Spark SQL s
 ```
 sh spark-sql --conf spark.sql.catalog.my_catalog=org.apache.iceberg.spark.SparkCatalog \
     --conf spark.sql.catalog.my_catalog.warehouse=s3://iceberg-warehouse/s3-tagging \
-    --conf spark.sql.catalog.my_catalog.catalog-impl=org.apache.iceberg.aws.glue.GlueCatalog \
+    --conf spark.sql.catalog.my_catalog.type=glue \
```

Comment:
there are more examples of `catalog-impl` in this file that should be adjusted

Reply:
I intentionally left it thinking examples can have both the kinds.

Let me update it then.

Reply:
updated and handled some more places + config docs.